src/components/CharacterAssignment.jsx

```diff
@@ -127,12 +128,7 @@ const CharacterAssignment = ({ actors, currentScriptId }) => {
   const addActorColumn = (actors, columns, columnOrder) => {
     actors.forEach((actor) => {
       if (!columns[actor.id]) {
-        columns[actor.id] = {
-          id: actor.id.toString(),
-          title: actor.name,
-          characterIds: [],
-          lineCount: 0
-        };
+        columns[actor.id] = new Column(actor.id.toString(), actor.name, characters);
```

Comment:
Since actor ids are plain numbers, it's possible that we could have some other id on the page with the same number and get a collision. Can we add a prefix/suffix to make it unique?

Reply:
Unless I'm reading this incorrectly and these won't ever be html ids...

Reply:
They will not ever be html ids. The column ids and character ids are both used for the drag-and-drop library. They are also set as component keys for React to do its diffing, but there won't be a collision there since the key is only for comparing different instances of the same component.

Ex: DropColumn.jsx
`<Droppable droppableId={column.id} key={column.id}>`

Ex: Character.jsx
`  <Draggable draggableId={character.id} index={index} key={character.id}>`

Reply:
Cool, thanks!